lib/src/typed_wrappers.dart

```diff
@@ -190,7 +178,6 @@ class TypeSafeList<E> extends TypeSafeIterable<E> implements DelegatingList<E> {
   int lastIndexOf(E element, [int start]) =>
       _listBase.lastIndexOf(element, start);
 
-  // TODO: Dart 2.0 requires this method to be implemented.
```

Comment:
You removed the TODO, but this is still unimplemented...

Reply:
Done, thanks!